src/index.tsx

```diff
@@ -137,5 +151,7 @@ serviceWorker.register()
     }
   })
   .catch((e) => {
-    Sentry.captureException(e);
+    if (!securityIsTooTight()) {
```

Comment:
i was thinking more like actually look at the exception and determine if it is the access denied error. doing a little testing in chrome i see that the exception doesn't look super standardized, i get a `DOMException` with message `"Failed to register a ServiceWorker for scope ('https://rex-web.herokuapp.com/books/') with script ('https://rex-web.herokuapp.com/books/service-worker.js'): The user denied permission to use Service Worker."`, so we'd probably have to do a check like `if (e instanceof DOMException && e.message.includes('user denied permission to use Service Worker'))`, which is a little sloppy but does what it says on its hat and is pretty straightforward. we'd need to test firefox/edge/safari to see if their error messages are similar though 

if we need to have a blocklist of exact error messages, like separate ones for each browser, it may become easier to put them [here](https://github.com/openstax/rex-web/blob/main/src/helpers/Sentry.ts#L34) so they don't clutter up the sentry code